src/librustc_trans/back/link.rs

```diff
@@ -288,7 +288,7 @@ pub fn mangle<PI: Iterator<Item=PathElem>>(path: PI,
     // when using unix's linker. Perhaps one day when we just use a linker from LLVM
     // we won't need to do this name mangling. The problem with name mangling is
     // that it seriously limits the available characters. For example we can't
-    // have things like &T or ~[T] in symbol names when one would theoretically
+    // have things like &T or Vec<T> in symbol names when one would theoretically
```

Comment:
Yeah, I wasn't sure either, but [`sanitize` does list `<` and `>`as characters that need to be escaped...](https://github.com/rust-lang/rust/blob/master/src/librustc_trans/back/link.rs#L242-L243) wdyt? Should I just take `or ~[T]` out? Is there a more appropriate second example?